src/hooks/useCampaignsHandler.ts

```diff
@@ -31,6 +33,39 @@ export const useCampaignsHandler = () => {
   const [campaigns, setCampaigns] = useState<ICampaign[]>([]);
 
   const addCampaign = async () => {
+    if (
```

Comment:
Can we put these into a separate validation function?

Reply:
Done